charts/refarch-gateway/templates/service-monitor.yaml

```
@@ -0,0 +1,20 @@
+  name: {{ include "refarch-gateway.fullname" . }}-monitor
+  labels:
+    {{- include "refarch-gateway.labels" . | nindent 4 }}
+spec:
+  endpoints:
+    - interval: 30s
+      port: http
+      scheme: http
+      path: /actuator/metrics
+  namespaceSelector:
+    matchNames:
+      - {{ .Release.Namespace }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)